components/Burger/Burger.tsx

```diff
@@ -3,12 +3,12 @@ import { StyledBurger } from "./style";
 
 interface IOpen {
   open: boolean;
-  setOpen: (value: boolean) => void;
+  isOpen: (value: boolean) => void;
 }
 
-export const Burger = ({ open, setOpen }: IOpen) => {
+export const Burger = ({ open, isOpen }: IOpen) => {
```

Comment:
it was correct bebore. You update is open state, do some action -> setOpen more clearable